demo/full/scripts/controllers/Main.tsx

```diff
@@ -17,7 +17,8 @@ function MainComponent(): JSX.Element {
             </a>
           </h1>
           <nav>
-            <a href="https://developers.canal-plus.com/rx-player/doc/Getting_Started/Welcome.html">
```

Comment:
Sad about this one, I suppose disabling eslint for that line is hard due to it being in JSX?